konch.py

```diff
@@ -345,11 +345,17 @@ def check_availability(self):
 
     def start(self):
         try:
-            from ptpython.repl import embed
+            from ptpython.repl import embed, run_config
         except ImportError:
             raise ShellNotAvailableError("PtPython shell not available.")
         print(self.banner)
-        embed(globals=self.context, vi_mode=self.ptpy_vi_mode)
+
+        if os.path.isfile(os.path.expanduser("~/.ptpython/config.py")):
+            embed(globals=self.context, vi_mode=self.ptpy_vi_mode, configure=run_config)
+
+        else:
+            embed(globals=self.context, vi_mode=self.ptpy_vi_mode)
```

Comment:
I still prefer the approach in https://github.com/sloria/konch/pull/51#issuecomment-444566663 because there's less duplication, and it's more in line with what ptpython does when running it "normally".

Having just one `embed(..)` call will make the code simpler when more arguments are passed (e.g. when/if we add support for history files).